src/Alarm.cpp

```diff
@@ -112,6 +112,7 @@ void Alarm::soundOneAlarmCycle(){
 }
 
 void Alarm::resetCalibration(){
+  Serial.write(14006); //14006 = Log, Info, Execute Command: Reset Calibration
```

Comment:
`006` isn't really appropriate here. Thats an execute command. It needs to have its own logging statement.